app/(auth)/login/_components/loginForm.tsx

```
@@ -57,52 +59,62 @@ export function LoginForm() {
-                                    fill="#4285F4"
-                                    d="M255.878 133.451c0-10.734-.871-18.567-2.756-26.69H130.55v48.448h71.947c-1.45 12.04-9.283 30.172-26.69 42.356l-.244 1.622l38.755 30.023l2.685.268c24.659-22.774 38.875-56.282 38.875-96.027"
-                                ></path>
+                            <div className="flex w-full items-center justify-center gap-2">
+                                <svg
+                                    xmlns="http://www.w3.org/2000/svg"
+                                    width="0.98em"
+                                    height="1em"
+                                    viewBox="0 0 256 262"
+                                    className="flex-shrink-0"
+                                    aria-hidden
+                                >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)